src/less/components/_data.less

```diff
@@ -63,7 +63,17 @@
 // Markup:
 //<p class="statusDataValue">
 //  <span class="active">
-//    Active
+//    Active Label
+//  </span>
+//</p> <br />
+//<p class="statusDataValue">
```

Comment:
Please check the task again. The task is to get rid off the statusDataValues and use the Bootstrap labels ( http://getbootstrap.com/components/#labels )